internal/controller/ws/client.go

```diff
@@ -196,6 +204,36 @@ func (c *rcClient) consume(ctx context.Context) {
 	}
 }
 
+func (c *rcClient) produce(ctx context.Context, m protocol.Message, kw *kafka.Writer) error {
```

Comment:
@Jason-RH I submitted a PR (PR #39) with some suggested changes to this method.  Please take a look and let me know what you think!  Let me know if we need to discuss.

Reply:
merged it in.

Reply:
Thanks!